src/state/megaStore.tsx

```diff
@@ -209,14 +217,41 @@ export const Provider: ParentComponent = (props) => {
             try {
                 if (state.mutiny_wallet && !state.is_syncing) {
                     setState({ is_syncing: true });
+                    let price: number;
+                    let currency: Currency;
                     const newBalance = await state.mutiny_wallet?.get_balance();
-                    const price =
-                        await state.mutiny_wallet?.get_bitcoin_price();
-                    setState({
-                        balance: newBalance,
-                        last_sync: Date.now(),
-                        price: price || 0
-                    });
+                    if (state.fiat.value === "BTC") {
+                        price = 1;
+                    } else {
+                        try {
+                            currency = JSON.parse(
+                                localStorage.getItem("fiat_currency") || ""
+                            );
+                            price =
+                                await state.mutiny_wallet?.get_bitcoin_price(
+                                    currency.value.toLowerCase() || "usd"
+                                );
+                            setState({
+                                balance: newBalance,
+                                last_sync: Date.now(),
+                                price: price || 0,
+                                fiat: currency
+                            });
+                        } catch (e) {
+                            price = 1;
+                            setState({
+                                balance: newBalance,
+                                last_sync: Date.now(),
+                                price: price,
+                                fiat: {
+                                    value: "BTC",
+                                    label: "bitcoin BTC",
```

Comment:
lowercase bitcoin here doesn't match the select option and it's the first thing you see when go to the chooser. could literally use `FIAT_OPTIONS[0]` if you wanted